keyboards/mode/m256wh/info.json

```diff
@@ -0,0 +1,110 @@
+{
+    "manufacturer": "Mode Designs",
+    "keyboard_name": "M256W-H",
+    "maintainer": "Gondolindrim",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
```

Comment:
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
```

Reply:
Committed in df17d397553eda8b5765c189eed4b988899b8878